packages/grid/x-data-grid/src/constants/gridClasses.ts

```diff
@@ -422,6 +422,19 @@ export interface GridClasses {
    * Styles applied to the toggle of the grouping criteria cell
    */
   groupingCriteriaCellToggle: string;
+  /**
+   * Styles applied to the pinned rows container.
+   */
+  pinnedRows: string;
+  /**
+   * Styles applied to the top pinned rows container.
+   */
+  'pinnedRows--top': string;
+  /**
+   * Styles applied to the bottom pinned rows container.
+   */
+  'pinnedRows--bottom': string;
+  pinnedRowsRenderZone: string;
```

Comment:
I guess this one is present on both top and bottom render zones right?

Reply:
Correct.
Do you think it's better to split it into two (top and bottom)?